docs/source/examples/predict_bike_trips.ipynb

```diff
@@ -0,0 +1,414 @@
+    "\n",
+    "### Defining the Labeling Process\n",
+    "\n",
+    "Let's start by defining a labeling function to calculate the number of trips. Given that each observation is an individual trip, the number of trips is just the number of observations."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def trip_count(ds):\n",
+    "    return len(ds)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Representing the Prediction Problem\n",
+    "\n",
+    "Then, let's represent the prediction problem by creating a label maker with the following parameters:\n",
+    "\n",
+    "* `target_entity` as the column for station ID where each trip starts from, since we want to process trips from each station.\n",
+    "* `labeling_function` as the function to calculate the number of trips.\n",
+    "* `time_index` as the column for the starting time of a trip. The biking periods are based on this time index.\n",
+    "* `window_size` as the length of a biking period. We can easily change this parameter to create variations of the prediction problem."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "lm = cp.LabelMaker(\n",
+    "    target_entity='from_station_id',\n",
+    "    labeling_function=trip_count,\n",
+    "    time_index='starttime',\n",
+    "    window_size='13h',\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Finding the Training Examples\n",
+    "\n",
+    "Now, let's run a search to get the training examples by using the following parameters:\n",
+    "\n",
+    "* The trips sorted by the start time.\n",
+    "* `num_examples_per_instance` to find the number of training examples per station. In this case, the search will return all existing examples.\n",
+    "* `minimum_data` as the start time of the first biking period. This is also the first cutoff time for building features."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "lt = lm.search(\n",
+    "    df.sort_values('starttime'),\n",
+    "    num_examples_per_instance=-1,\n",
+    "    minimum_data='2014-06-30 08:00',\n",
+    "    verbose=False,\n",
+    ")\n",
+    "\n",
+    "lt.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The output from the search is a label times table with three columns:\n",
+    "\n",
+    "* The station ID associated to trips.\n",
+    "* The start time of the biking period. This is also the cutoff time for building features. Only data that existed beforehand is valid to use for predictions.\n",
+    "* The number of trips. This is calculated by our labeling function.\n",
```

Comment:
I would say "The number of trips during the biking period window." to be more clear that this isn't the total number of trips from the station - only the number during the window. Also might be good to mention we can get multiple entries for a single station if the data spans multiple windows (station id 181 for example).